lib/facter/self_service.rb

```diff
@@ -108,6 +108,13 @@
     { S0013: File.open(summary_path).read.include?('catalog_application') }
   end
 
+  chunk(:S0019) do
+    next unless PuppetSelfService.primary? || PuppetSelfService.replica? || PuppetSelfService.compiler? || PuppetSelfService.legacy_compiler?
+    pupserv_api = PuppetSelfService.status_check('8140', 'pe-jruby-metrics?level=debug')
+    jruby_check = pupserv_api['status']['experimental']['metrics']['average-free-jrubies']
+    { S0019: jruby_check > 1 }
```

Comment:
```suggestion
    { S0019: jruby_check > = 1 }
```

This will prevent the default "true" from not triggering in the test environment ( i hope)